ddtrace/contrib/pytest/plugin.py

```diff
@@ -250,7 +251,8 @@ def _start_test_suite_span(item, test_module_span, should_enable_coverage=False)
     test_suite_span.set_tag_str(test.FRAMEWORK_VERSION, pytest.__version__)
     test_suite_span.set_tag_str(test.COMMAND, _get_pytest_command(item.config))
     test_suite_span.set_tag_str(_EVENT_TYPE, _SUITE_TYPE)
-    test_suite_span.set_tag_str(_SESSION_ID, str(test_session_span.span_id))
+    if test_session_span:
```

Comment:
It seems possible that the changes in this file fix a bug that isn't directly related to adding unittest support. It might be beneficial to pull them out into a separate pull request.

Reply:
Good catch, will draft a quick PR for it